homeassistant/components/daikin/__init__.py

```diff
@@ -53,7 +53,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: DaikinConfigEntry) -> bo
             )
         _LOGGER.debug("Connection to %s successful", host)
     except TimeoutError as err:
-        _LOGGER.debug("Connection to %s timed out in 60 seconds", host)
+        _LOGGER.debug("Connection to %s timed out in 120 seconds", host)
```

Comment:
```suggestion
        _LOGGER.debug("Connection to %s timed out in %s seconds", host, TIMEOUT_SEC)
```
therefore the next time someone changes that, it will not be forgotten